R/preprocess.R

```diff
@@ -28,4 +28,6 @@ create_network <- function(data, crs = NULL) {
   if (!is.null(crs)) {
     net <- net |> sf::st_transform(crs)
   }
+
+  net
```

Comment:
This function was not returning the output - a small fix outside the scope of the PR to enable tests on this function to run

Reply:
I do not think this is necessart though. Did you have issues with this? See my comment below.

Reply:
Uhm, however here the function does not return the expected output unless the crs (optional) argument is specified. Maybe the behaviour is different with the if-block closing the function?

See following example:

```r
f <- function(x, opt = FALSE) {
  y <- x + 1

  if (opt) {
    y <- y + 1
  }
} 

a <- f(1)  # a is NULL
a <- f(1, opt = TRUE)  # a is 3 
```

Reply:
Indeed. It is different if the body of the function ends with an if block. Then let's keep it as you suggest. Thanks.